drivers/storage/portworx/portworx.go

```diff
@@ -595,6 +595,9 @@ func (p *portworx) SetDefaultsOnStorageCluster(toUpdate *corev1.StorageCluster)
 				pxVersionChanged ||
 				autoUpdateComponents(toUpdate)) {
 			toUpdate.Status.DesiredImages.Stork = release.Components.Stork
+			if toUpdate.Status.DesiredImages.KubeScheduler == "" {
```

Comment:
Can we add a couple of UTs for this condition and when stork is disabled?
The existing UTs reside in this function - https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/portworx_test.go#L2683. You can just add the sub-cases under this.